cypress/components/multi-action-button/multi-action-button.cy.tsx

```diff
@@ -558,9 +574,6 @@ context("Tests for MultiActionButton component", () => {
     it("should have the expected border radius and focus styling on main button", () => {
```

Comment:
```suggestion
    it("should have the expected border radius on main button", () => {
```

**comment**: we're no longer testing the focus styling in this test so keeping the old description is potentially confusing

Reply:
Changed as per comment